src/main/java/com/google/research/bleth/utils/Queries.java

```diff
@@ -116,4 +117,25 @@ public static double average(List<Entity> entities, String property) throws Clas
         return properties.stream()
                 .collect(toImmutableMap(Function.identity(), property -> average(entities, property)));
     }
+
+    /**
+     * Given a simulationId as a string, delete all data associated with that simulation id.
+     * @param simulationId is the simulation id.
+     */
+    public static void delete(String simulationId) {
```

Comment:
Does DataStore support transactions? If so, can you wrap all these deletions in a single transaction (so a partial failure won't leave our database in a weird state)?